dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -816,6 +818,15 @@ def parallel_test_types_list_as_string(self) -> str | None:
         self._extract_long_provider_tests(current_test_types)
         return " ".join(sorted(current_test_types))
 
+    @cached_property
+    def providers_test_types_list_as_string(self) -> str | None:
+        all_test_types = self.parallel_test_types_list_as_string
+        if all_test_types is None:
+            return None
+        return " ".join(
+            [test_type for test_type in all_test_types.split(" ") if test_type.startswith("Providers")]
+        )
```

Comment:
```suggestion
        return " ".join(
            test_type for test_type in all_test_types.split(" ") if test_type.startswith("Providers")
        )
```

Reply:
Ah nice. Indeed. I wonder if there is a ruff rule for that ?